portfolio/src/main/webapp/script.js

```diff
@@ -30,9 +30,12 @@ function addRandomFact() {
 function fetchComments() {
     fetch("/data").then(request => request.json()).then((textResponse) => {
         for (var i = 0; i < textResponse.length; i++) {
-            var paragraph_tag = document.createElement("p");
-            paragraph_tag.innerText = textResponse[i]['comment'];
-            document.getElementById("comments-container").appendChild(paragraph_tag);
+            var paragraphTag = document.createElement("p");
```

Comment:
Now that we have two separate paragraphs, may be better variable names for the two elements would be "commentParagraph" and "timestampParagraph", instead of using the same name "paragraphTag". What do you think?

Reply:
I updated the variable names.